checkov/secrets/plugins/entropy_keyword_combinator.py

```diff
@@ -157,12 +163,14 @@ def analyze_iac_line_yml(
             raw_context: CodeSnippet | None = None,
             value_pattern: dict[Pattern[str], int] | None = None,
             secret_pattern: dict[Pattern[str], int] | None = None,
+            multiline_parser: BaseMultiLineParser | None = None,
```

Comment:
@marynaKK This should never be null here

Reply:
If this is null - no multiline is declared for this type of object. Is it not a valid approach? How can I allow the case where there is no multiline parser?

Reply:
This method is called only if `multiline_parser` is not `None` I believe